src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -294,6 +267,7 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
 
   /** Creates the scenario and the uploaded project areas, if provided. */
   createScenarioAndProjectAreas(): void {
+    this.generatingScenario = true;
     this.createUploadedProjectAreas()
```

Comment:
I'm guessing that replacing this is the work for your next EP.

Reply:
This boolean actually controls whether the generate button is enabled (it's disabled upon scenario creation to keep users from submitting multiple requests). Replacing createScenario in planService will be in the pr after next (adding in maxArea)